nodes/consensus-node.md

```diff
@@ -151,180 +151,63 @@ alias head=ghead
 
 ## Storage and pruning configurations
 
-### Recommendations per node type
+### Connecting a consensus node to a bridge node
 
-Here are the summarized recommendations for each node type. There are more details on what each of these settings do after the recommendations. Understanding what these settings do will help you make the best decision for your setup. Note that all of these settings can be modified in the config files directly or by using the their respective flags that use the same name.
+If your consensus node is being connected to a celestia-node bridge node,
+you will need to enable transaction indexing and retain all block
+data. This can be achieved with the following settings in your `config.toml`.
 
-#### Validator node
-
-The recommendations here are assuming that the validator node is isolated from other responsiblities other than voting and proposing. This means that it is not indexing transaction, storing the results of the execution of txs, and it's only storing the past two state snapshots. Note that if the validators are connected to a bridge node then the [serving a bridge node](#consensus-node-serving-a-bridge-node) configuration should be used.
-
-`config.toml`:
+#### Enable transaction indexing
 
 ```toml
-indexer = "null"
-```
-
-`app.toml`:
-
-```toml
-# by setting custom and a keep recent value of 2, the node will only
-# keep the past two state snapshotss
-pruning = "custom"
-pruning-keep-recent = "2"
-pruning-interval = "10"
-snapshot-interval = 1500
-```
-
-#### RPC node
-
-RPC nodes are optimized to be useful for querying onchain data at the cost of significantly increased storage requirements. This means storing all block data, indexing all transactions and the results of their execution, and store the past 3 weeks of state snapshots.
-
-`config.toml`:
-
-```toml
-indexer = "kv" # or "psql"
-```
-
-`app.toml`:
-
-```toml
-min-retain-blocks = 0
+indexer = "kv"
```

Comment:
let's keep this for now, but I think there was a miscommunication somewhere. I was under the impression that this was needed for bridge nodes, but after looking through the code I don't think it is. There's no time to properly test if it is or not, so I think its best to keep this as is for now.

Reply:
understood, thank you for the context.